modules/cluster-logging-rn-5.4.4.adoc

```diff
@@ -0,0 +1,18 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-4"]
+= Logging 5.4.4
+This release includes link:https://access.redhat.com/errata/RHBA-2022:5907[RHBA-2022:5907-OpenShift Logging Bug Fix Release 5.4.4].
+
+[id="openshift-logging-5-4-4-bug-fixes"]
+== Bug fixes
+
+* Before this update, non-latin characters displayed incorrectly in Kibana. With this update, Kibana displays all valid UTF-8 symbols correctly. (link:https://issues.redhat.com/browse/LOG-2794[LOG-2794])
+
+* Before this update, the `cluster-logging` Operator relied on the cluster to create a secret. This cluster behavior changed in {product-title} 4.11, which caused logging deployments to fail. With this update, the `cluster-logging` Operator resolves the issue by creating the secret if needed. (link:https://issues.redhat.com/browse/LOG-2840[LOG-2840])
+
```

Comment:
@libander Could you check if the below bugs for 5.4.4 need no-rn label? They are in errata but not in docs.
https://issues.redhat.com/browse/LOG-2657 
https://issues.redhat.com/browse/LOG-2821